spec/helpers/feedbacks_helper_spec.rb

```diff
@@ -46,6 +46,17 @@ class Klass
         klass.open_requests(user).should == 5
       end
     end
+
+    describe 'unsubmitted external feedback' do
+      let(:ac_user) { create(:user) }
+      let(:ac) { create(:associate_consultant, :user => ac_user) }
+      let!(:feedback) { create(:feedback, user: ac_user) }
+
+      it 'should not include external feedback in the count of open requests' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.